pkg/bbgo/strategy_controller.go

```diff
@@ -0,0 +1,119 @@
+package bbgo
+
+import (
+	"fmt"
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	"reflect"
+)
+
+type StrategyController struct {
+	Status types.StrategyStatus
+
+	// Callbacks
+	GetStatusCallback     func() types.StrategyStatus
+	GetPositionCallback   func() *types.Position
+	ClosePositionCallback func(percentage fixedpoint.Value) error
+	SuspendCallback       func() error
+	ResumeCallback        func() error
+	EmergencyStopCallback func() error
+}
```

Comment:
OnSuspend, OnResume can be done by the callbackgen.

Reply:
the controller is simplified now
we dont need to use calbackgen?

Reply:
use callbackgen thanks 😊 more consistent

Reply:
done